app/controllers/email_confirmation_controller.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+class EmailConfirmationController < ApplicationController
+  def verify
+    @title = I18n.t('confirmation_mailer.title')
+    @member = Member.find_by(email: params[:email])
+
+    errors = EmailVerifierService.verify(params[:token], params[:email], cookies)
+
+    @rendered = template.render(
+      'errors' => errors,
+      'members_dashboard_url' => Settings.members.dashboard_url
+    ).html_safe
+
+    render 'email_confirmation/follow_up', layout: 'generic'
+  end
+
+  private
+
+  def template
+    ## FIXME seed and fetch from DB
+    #
+    view = File.read("#{Rails.root}/app/liquid/views/layouts/email-confirmation-follow-up.liquid")
```

Comment:
We have two different templating engines right now that we use for different things. One is to allow Champaign users to edit the appearance and functionality of campaign pages. The other is for templates that are required in the codebase for the code to work, and is much more concise and makes it harder to make mistakes. In my mind, liquid templates should only be served through the LiquidRenderer, and no controller action should be dependent on a specific liquid template's existence in order to do its job.